pkg/cmd/server/start/start_master.go

```diff
@@ -245,6 +253,8 @@ func (o MasterOptions) RunMaster() error {
 		return nil
 	}
 
+	// Inject feature flags.
+	masterConfig.FeatureConfig = o.FeatureConfig
```

Comment:
Using the `[]string` to store `DisabledFeatures` will allow you to simply append the forbidden features right here instead of doing a straight replacement.  This will preserve an admin's intent if they actually want to disable something.